evaluator/evaluate.go

```diff
@@ -257,8 +258,9 @@ func loadPackages(tx *gorm.DB, accountID, systemID int,
 	err := tx.Table("package").
 		Select("concat(pn.name, '-', package.evra) nevra, package.*, sp.*").
 		Joins("join package_name pn on package.name_id = pn.id").
-		Joins(`left join system_package sp on sp.package_id = package.id`).
-		Where("sp.rh_account_id = ? AND sp.system_id = ?", accountID, systemID).
+		// We need to perform left join, so thats why the parameters are here
```

Comment:
What to add some tests to reveal issues like this?

Reply:
Added more checks to `TestEvaluate`

Reply:
good, thanks